### PR TITLE
[QC-570] script to release

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: Framework/script/release.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+fi
+#git push
+
+
+read -p "Go to JIRA and release this version : https://alice.its.cern.ch/jira/projects/QC?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=released-unreleased
+Press any key to continue" -n 1 -r
+echo
+
+
+echo "Go to Github https://github.com/AliceO2Group/QualityControl/releases/new?tag=v${newVersion}&title=v${newVersion}&body=$(cat ${scriptDir}/../../doc/ReleaseNotesTemplate.md |jq -sRr @uri)"
```
Comment (knopers8) [inline Framework/script/release.sh:88]: I didn't have `jq` on Ubuntu. Could you check if it is installed at the beginning?

File: Framework/script/release.sh [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+git commit -m "${newVersion}"
+
+
+read -p "Ready to push, please confirm [y|n]" -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    exit 3
+fi
+#git push
```
Comment (knopers8) [inline Framework/script/release.sh:80]: I tihnk the push is actually missing here

**Review by knopers8 — COMMENTED** [review]
Nice, I like it! Please have a look at my minor suggestions.

File: Framework/script/release.sh (hunk 2 above)
Comment (Barthelemy) [reply Framework/script/release.sh:80]: ah yes, I did not want to mess up too much during development. Thank you

File: Framework/script/release.sh (hunk 1 above)
Comment (Barthelemy) [reply Framework/script/release.sh:97]: done

**Review by knopers8 — APPROVED** [review]

**Pull request merged**